run_meson_command_tests.py

```diff
@@ -212,6 +212,7 @@ def test_meson_runpython(self):
             mesonout = self._run(python_command + [meson_command, 'runpython'] + cmd, env=env)
             self.assertEqual(pyout, mesonout)
 
+        os.remove(script_file)
```

Comment:
To ensure this gets cleaned up even if there's a failure, add `self.addCleanup(os.remove, scriptfile)` righta fre the `with open(...):` block. That cleanup happens regardless of exceptions